README.md

```diff
@@ -80,8 +80,8 @@ Even with these basic mechanics, there is quite a lot of complexity in creating
     * All platforms: VT-x/AMD-v virtualization.
     * Linux: [Virtualbox](https://www.virtualbox.org/wiki/Downloads).
     * OSX: either [Virtualbox](https://www.virtualbox.org/wiki/Downloads) or [VMWare Fusion](http://www.vmware.com/products/fusion.html).
+* Download Docker, Minikube, and Kubectl before running the script.
```

Comment:
can we make it easier for people that are working locally though?

Reply:
I think this is the command you can run for mac providing you have homebrew installed: 
`brew update && brew install kubectl && brew cask install docker minikube virtualbox`

Reply:
Done for Mac. Can't do it for other platforms as I can't test

Reply:
Done 👍